src/__test__/test-helper.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Please rename this file to `testHelper` for consistency. We use camelCasing, not kebab-casing.